### PR TITLE
Fix discovery page 'supportedValues' failures

### PR DESCRIPTION
**Pull request by paulineribeyre**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)

### Bug Fixes
- Fix discovery page errors when `features.authorization` is enabled but `features.authorization.supportedValues` is not configured

### CODE REVIEW

File: docs/portal_config.md [hunk 1]
```diff
@@ -458,7 +458,8 @@ Below is an example, with inline comments describing what each JSON block config
         "enabled": true
       },
       "authorization": {
-        "enabled": true // toggles whether Discovery page displays users' access to studies. If true, 'useArboristUI' must also be set to true.
+        "enabled": true, // toggles whether Discovery page displays users' access to studies. If true, 'useArboristUI' must also be set to true.
+        "supportedValues": ["accessible", "unaccessible", "pending", "notAvailable"] // no default; should be configured if `authorization.enabled=true`
```
Comment (mfshao) [inline docs/portal_config.md:462]: this looks like 

```
"supportedValues": {
          "accessible": {
            "enabled": true,
            "menuText": "Available"
          },
         "unaccessible": {
            "enabled": false,
            "menuText": "Not Accessible"
          },
         ...
}
```

Comment (paulineribeyre) [reply docs/portal_config.md:462]: oh right 🤦‍♀️ thanks

File: docs/portal_config.md [hunk 2]
```diff
@@ -458,7 +458,19 @@ Below is an example, with inline comments describing what each JSON block config
...
+        "supportedValues": { // no default; should be configured if `authorization.enabled=true`
+          "accessible": {
+            "enabled": true,
+            "menuText": "Available"
+          },
+          "unaccessible": {
+            "enabled": false,
+            "menuText": "Not Accessible"
+          },
+          "unaccessible": {...},
```
Comment (mfshao) [inline docs/portal_config.md:471]:
```suggestion
          "pending": {...},
```
?

**Review by mfshao — APPROVED** [review]
thanks 🎉

**Pull request merged**